rfc/0007-community-team.md

```diff
@@ -233,7 +233,7 @@ The election has **failed** when one of the following is true:
 * A week has passed and less than 75% of the community team and administrative board has voted
 * A week has passed, at least 75% of the community team and administrative board has voted, and those that have
   voted have reached a consensus in opposition of the candidate - or a consensus cannot be reached
-* The candidate receives more negative votes than positive votes
+* An absolute majority of the community team and administrative board has voted against the candidate
```

Comment:
I would reword this.

> The candidate recieves enough negative votes to outweigh the remaining count of possible positive votes

That's the intention of this point, and I think this wording captures the style I used in the rest of the RFC better.

Reply:
In my opinion, `the remaining count of possible positive votes` is a bit too hard to digest, and can be interpreted as "the count of votes that could possibly be positive, but aren't yet" (because `possible` has the connotation of 'able to be done, but not yet done').

May I suggest the alternative of:
> The candidate recieves more negative votes than the total amount of positive votes and votes not yet submitted

Reply:
I guess that's fine too, yeah.

Reply:
Done.

Reply:
Tbh I find this more confusing than the original.

Reply:
How do you figure?

`negative > (positive + undecided)`

Reply:
Using a mathematical expression seems weird. 

I checked on Wikipedia where it mentions that "absolute majority" often means a majority of all those who can vote, and not just all those who have voted, but that this definition is not always used. 

But because of the fact it says "Absolute majority of the community team and administrative board", there shouldn't be any confusion possible.

Reply:
I don't really think that's a necessary clarification, especially given that this is an internal process for the most part - it's important that the RFC accurately describes the process, but I don't think it necessarily needs to completely avoid every chance of needing to be clarified for some people.

We should avoid bikeshedding this.